hasher-matcher-actioner/terraform/main.tf

```diff
@@ -239,6 +239,8 @@ module "api" {
   config_table          = local.config_table
   measure_performance   = var.measure_performance
   te_api_token_secret   = aws_secretsmanager_secret.te_api_token
+
```

Comment:
nit: random new line?

Reply:
Added that to separate from the ones above

Reply:
again complete nit but what separates this variables from the ones above, the previous 5 lines/vars seem like a random collection?